src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -104,24 +94,26 @@ class DraftJSEditor extends React.Component<Props> {
 
         return (
             <div className={classes}>
-                <Label hideLabel={hideLabel} showOptionalText={!isRequired} text={label}>
-                    <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
-                        {/* need div so tooltip can set aria-describedby */}
-                        <div>
-                            <Editor
-                                {...a11yProps}
-                                editorState={editorState}
-                                handleReturn={this.handleReturn}
-                                onBlur={handleBlur}
-                                onChange={handleChange}
-                                onFocus={onFocus}
-                                placeholder={placeholder}
-                                readOnly={isDisabled}
-                                stripPastedStyles
-                            />
-                        </div>
-                    </Tooltip>
-                </Label>
+                <span className={classNames({ 'accessibility-hidden': hideLabel })} id={this.labelID}>
```

Comment:
I think that `showOptionalText` controls an indicator whether the form field is required or not: `My Text (optional)`

Reply:
I thought it was indicating whether the optional field was required?  `Whether to show the `(Optional)` text next to the label for an optional field`

Is this needed? @alexkrolick

Reply:
Yes, it adds text next to the label that says `(Optional)` (localized string). That was handled by the Label component but it's not here any more.

Reply:
👍